src/management-system-v2/lib/helpers.ts

```diff
@@ -194,14 +199,19 @@ async function updateProcessVersionBasedOn(processDefinitionsId: string, version
   });
 
   if (processInfo?.bpmn) {
-    const versionInformation = await getDefinitionsVersionInformation(processInfo.bpmn);
-    const bpmn = await setDefinitionsVersionInformation(processInfo.bpmn, {
-      ...versionInformation,
-      versionBasedOn,
-    });
-    await put('/process/{definitionId}', {
-      params: { path: { definitionId: processDefinitionsId } },
-      body: { bpmn },
-    });
+    const { version, description, name } = await getDefinitionsVersionInformation(processInfo.bpmn);
+
+    if (version) {
```

Comment:
Why was this condition added? Since this works on the editable version of the process which should not have version info except `versionBasedOn` this condition should always be false, which would prevent updating the `versionBasedOn` value